packages/flutter/lib/src/rendering/sliver.dart

```diff
@@ -957,6 +965,15 @@ class SliverPhysicalParentData extends ParentData {
   /// top left visible corner of the sliver.
   Offset paintOffset = Offset.zero;
 
+  /// The [flex] factor to use for this sliver child.
+  ///
+  /// If null or zero, the child is inflexible and determines its own size.
+  /// If non-zero, the amount of space the child can occupy in the
+  /// cross axis is determined by dividing the free space (after placing the
+  /// inflexible children) according to the flex factors of the flexible
+  /// children.
+  int? flex = 1;  
```

Comment:
Just had a thought, can you name this crossAxisFlex in case there is a future where we flex for some reason in the main axis (also avoids confusion over what flexes here).